src/Symfony/Bridge/Doctrine/Security/RememberMe/DoctrineTokenProvider.php

```diff
@@ -40,11 +41,13 @@
  */
 class DoctrineTokenProvider implements TokenProviderInterface
 {
-    private $conn;
+    private const TABLE = 'rememberme_token';
 
-    public function __construct(Connection $conn)
+    private $connection;
```

Comment:
renaming the private property will increase the conflicts between branches. So it should either be done in 4.4 or not be done IMO.

Reply:
Ok, reverted